src/sagas/wallet.js

```diff
@@ -447,6 +448,43 @@ export function* listenForWalletReady(wallet) {
   }
 }
 
+/**
+ * An abstraction effect to call wallet's checkAddressMine method.
+ * This effect applies a retry strategy and throws a fatal error
+ * after max retry attempts.
+ *
+ * @param {Object} wallet Either fullnode or wallet-service wallet.
+ * @param {string[]} txAddresses A collection of addresses to check.
+ * @returns {{ [address: string]: boolean }} A map of type address:isMine.
+ */
+function* checkAddressMine(wallet, txAddresses) {
+  const maxRetries = 5;
```

Comment:
suggestion: Make this and the increase factor below constants

Reply:
Done at https://github.com/HathorNetwork/hathor-wallet-mobile/pull/444/commits/89c53c7f1c73034086e9cc5e4f59375df32fcf26